src/core/quantum_circuit.jl

```diff
@@ -453,7 +453,7 @@ end
 function get_split_circuit_layout(io::IO, circuit_layout::Array{String},
     padding_width::Integer)
 
-    (display_height, display_width) = displaysize(io)
+    (display_height, display_width) = displaysize(stdout)
```

Comment:
this only affects the doctests, in which case there's a difference between the outputs of `displaysize(io)` and `displaysize(stdout)`, the former always returning the default width of 80. The other instances of `::IO` are `stderr` and `stdin`, which won't be passed by `Base.show()`

Reply:
`IO` is an interface and can be much more than just `stderr` and `stdin`. It can even be user-defined, and we can easily get an `IO` type with another `displaysize` which will render incorrectly with base types.

```julia
    buffer = IOBuffer(UInt8[], read=true, write=true)
    io = IOContext(:displaysize = (30, 120))
```

We really should use the `displaysize` of the `IO` type. If the doctests are inconsistent with their width then we need to find a way to fix the inconsistency. If the doctests are consistent and always use the same width then the output of the doctests should be updated to match the consistent behaviour of doctests.